config/buildAssets.js

```diff
@@ -1,69 +1,70 @@
 const fs = require('fs/promises');
 const path = require('path');
 const esbuild = require('esbuild');
-const { sassPlugin } = require('esbuild-sass-plugin');
+const {sassPlugin} = require('esbuild-sass-plugin');
 
 async function createAssetPaths() {
-  let pathPrefix = ''
+    let pathPrefix = ''
 
-  if (process.env.BASEURL) {
-    pathPrefix = process.env.BASEURL
-  }
+    if (process.env.BASEURL) {
```

Comment:
We don't have any documentation on it in this repo but our IDE's and linting processes are configured to save `JS` indents as `spaces` with the size of `2`. Update your IDE and resave the `JS` files. Sorry for the confusion.

Reply:
It probably makes the most sense to address this on this request all at once since a lot more content has been migrated since this PR. https://github.com/cloud-gov/site/pull/9